biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/AbstractFeature.java

```diff
@@ -271,13 +271,13 @@ public void setUserObject(Object userObject) {
     }
     
 	@Override
-	public HashMap<String, Qualifier> getQualifiers() {
+	public HashMap<String, ArrayList<Qualifier>> getQualifiers() {
```

Comment:
The core-module does currently not have any dependencies on external libraries and I'd prefer to keep it that way. Only the genomics module uses Guava. as such my vote is for 
```java
public Map<String, List<Qualifier>> getQualifiers() { ... }
```

(note: use of interfaces, not specific implementations)